via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -169,10 +172,26 @@ export default function VideoPlayerApp({
   }, [nextRender]);
 
   const syncTranscript = useCallback(
-    () => transcriptControls.current!.scrollToCurrentSegment(),
+    () => transcriptControls.current?.scrollToCurrentSegment(),
```

Comment:
This avoids an error if the user manages to set and then clear a filter before the transcript is loaded. We could try to disable the filter input if the transcript has not yet loaded, but given that there are a number of triggers that clear the filter, I think it is going to be more reliable to check when this function is called.